executor/aggregate_xapi.go

```diff
@@ -298,7 +298,8 @@ func (e *NewXAggregateExec) Next() (*Row, error) {
 		e.aggregaters = make([]*finalAggregater, len(e.AggFuncs))
 		for i, af := range e.AggFuncs {
 			agg := &finalAggregater{
-				name: strings.ToLower(af.GetName()),
+				name:               strings.ToLower(af.GetName()),
+				contextPerGroupMap: make(map[string](*ast.AggEvaluateContext)),
```

Comment:
So we can drop the check: https://github.com/pingcap/tidb/blob/master/executor/aggregate_xapi.go#L59